chrome/options.browserify.js

```diff
@@ -9,6 +9,11 @@ var settings = {
     title: "Use fuzzy search",
     value: true
   },
+  filterEverything: {
+    type: "checkbox",
+    title: "Filter all available entries",
```

Comment:
The name `filterEverything` makes sense to you and me, because we know how it works under the hood, so I would keep the naming `filterEverything` everywhere. But as for label, I would probably switch to something like `Search passwords in real time`, which should be more intuitive for users on what to expect when they check this option 🙂